static/app/views/settings/components/forms/selectField.tsx

```diff
@@ -104,7 +104,7 @@ export default class SelectField<
                 multiple={multiple}
                 onChange={val => {
                   const previousValue = props.value?.toString();
-                  const newValue = val.value?.toString();
+                  const newValue = val?.value?.toString();
```

Comment:
Solves [JAVASCRIPT-25GA]https://sentry.io/organizations/sentry/issues/2622603673/)

Reply:
It would happen when removing the value of an optional field